packages/fury-adapter-swagger/test/fixtures/info-termsOfService.json

```diff
@@ -0,0 +1,44 @@
+{
+  "element": "parseResult",
+  "content": [
+    {
+      "element": "category",
+      "meta": {
+        "classes": {
+          "element": "array",
+          "content": [
+            {
+              "element": "string",
+              "content": "api"
+            }
+          ]
+        },
+        "title": {
+          "element": "string",
+          "content": "Sample Pet Store App"
+        }
+      },
+      "attributes": {
+        "version": {
+          "element": "string",
+          "content": "1.0.1"
+        }
+      },
+      "content": [
+        {
+          "element": "link",
+          "attributes": {
+            "relation": {
+              "element": "string",
+              "content": "terms-of-service"
```

Comment:
Terms of service don't convey transactional semantics of an API. Wouldn't it better to add add it as a meta entry, alongside `title`?

Also, perhaps we should standardize this feature in https://github.com/apiaryio/api-elements before we merge an implementation.

Reply:
with the next commit it will be under `link`, see if it is ok for you

Reply:
Perfect.